src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
I've played around a bit now and here are some thoughts.

#### Mixing inline styles and Aphrodite: 

I was not able to get this to work 100%.  There was some cognitive over head around thinking through how inline styles over write styles in the Aphrodite injected style sheet.  I'm sure we could probably get it to work with some more effort but more complex components might end up being kinda of weird to follow.

#### Dynamic/Static styles all with Aphrodite

This works great with no issues.  The only down side to this was that we had two or more sets of styles.  While not terrible, I like having all the styles for a component together in one location but that is personal preference.  This still requires a delay or setTimeout.

#### Leave as is with a small change to using setTimeout

In the end I still like this approach best.  I've moved to using setTimeout over delay via Velocity and I've also used zero for the time out delay.  I looked into doing a decorator as Cameron suggested but I was coming up short with a good way to do it and in the end it seemed over kill.  I'm open to sitting down with Cameron next week for an education on decorators to see if we can come up with something better than I could.